match/lib/match/nuke.rb

```diff
@@ -172,15 +172,15 @@ def delete_files!
     def certificate_type(type)
       cert_type = Spaceship.certificate.production
       cert_type = Spaceship.certificate.development if type == :development
-      cert_type = Spaceship.certificate.in_house if Match.enterprise? && Spaceship.client.in_house?
+      cert_type = Spaceship.certificate.in_house if type == :enterprise
```

Comment:
I assume that's because _spaceship_ uses the official names being used by the Apple Developer Portal. I don't feel like it's something we need to change, from the user's perspective it seems pretty solid.